modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -53,31 +54,45 @@
         />
       </div>
       <div class="apos-admin-bar__row">
-        <span class="apos-admin-bar__context-spacer" />
-        <span class="apos-admin-bar__context-title">
-          <information-outline-icon />
+        <div class="apos-admin-bar__context-spacer" />
+        <div class="apos-admin-bar__context-title">
+          <span
+            v-tooltip="'Page Title'" class="apos-admin-bar__context-title__icon"
+          >
+            <information-outline-icon fill-color="var(--a-primary)" :size="16" />
+          </span>
           {{ moduleOptions.context.title }}
-        </span>
-        <span class="apos-admin-bar__context-controls">
+        </div>
+        <div class="apos-admin-bar__context-controls">
           <AposButton
             v-if="editMode"
-            type="default" label="Preview Mode"
-            icon="eye-icon" class="apos-admin-bar__btn"
-            :icon-only="true"
+            class="apos-admin-bar__context-button"
+            label="Preview Mode" :tooltip="{
+              content: 'Preview Mode',
+              offset: 0,
+              placement: 'bottom'
+            }"
+            type="outline" :modifiers="['no-motion']"
+            icon="eye-icon" :icon-only="true"
             @click="switchToPreviewMode"
           />
           <AposButton
             v-if="!editMode"
-            type="default" label="Edit Mode"
-            icon="pencil-icon" class="apos-admin-bar__btn"
-            :icon-only="true"
+            class="apos-admin-bar__context-button"
+            label="Edit" icon="pencil-icon"
+            :modifiers="['no-motion']"
```

Comment:
Pencil icon really showing off how not-centered the label and icon are
![image](https://user-images.githubusercontent.com/1889830/99474353-6b3d4c80-291a-11eb-9e0c-1b7335266f16.png)

Reply:
I also approve.

Reply:
For some reason it's not looking as off in my Chrome. 
![Screen Shot 2020-11-18 at 11 32 29 AM](https://user-images.githubusercontent.com/1155984/99565854-c2cacf00-2991-11eb-83a2-23fe1614ed74.png)